src/app/student-dashboard/student-dashboard.component.ts

```diff
@@ -1,16 +1,33 @@
 import { Component, Input, OnInit } from '@angular/core';
-import { ActivatedRoute, RouterModule, Routes, Router, ParamMap } from '@angular/router';
+import { ActivatedRoute, RouterModule, Routes, Router, ParamMap, Params } from '@angular/router';
+import { Student } from '../student/student';
+import { StudentService } from '../student/student.service';
+import 'rxjs/add/operator/switchMap';
+import { Observable } from 'rxjs/Observable';
+
 
 @Component({
   selector: 'app-student-dashboard',
   templateUrl: './student-dashboard.component.html',
   styleUrls: ['./student-dashboard.component.css']
 })
 export class StudentDashboardComponent implements OnInit {
+  student$ : Observable<Student>;
```

Comment:
Does the `$` have a specific function here or just aesthetic.  I'm okay with it being here just curious.